src/Analyzers/CSharp/Tests/AbstractBuiltInCodeStyleDiagnosticAnalyzer/AbstractBuiltInCodeStyleDiagnosticAnalyzerTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.AbstractBuiltInCodeStyleDiagnosticAnalyzer;
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+using Roslyn.Utilities;
+using Xunit;
+
+public sealed class AbstractBuiltInCodeStyleDiagnosticAnalyzerTests
```

Comment:
>AbstractBuiltInCodeStyleDiagnosticAnalyzerTests [](http://example.com/codeflow?start=20&length=47)

Didn't see any tests for this class, so I added a single one to get things started for it that simply tests that the SupportedDiagnostics we send them have their order maintained.